casestudies/load-testing/README.md

```diff
@@ -0,0 +1,70 @@
+# ACK Performance at Scale
+
+In this usecase, we would deploy 10K SNS Topics, 20K SQS Queues and 20K SNS Subscriptions i.e 50K resources at a time using SQS and SNS ACK Controllers to examine performance of ACK and other metrics.
+
+**Testing environment:**
+* EKS cluster
+* ACK SNS and AWS SQS Controller
+* Prometheus + Grafana Stack
```

Comment:
Can you also provide the EKS Cluster version and ACK controller versions? Maybe talk about the instance types/arch you're using?

Reply:
Updated with cluster and controller versions